tests/metal/modules/test_lstm.py

```diff
@@ -84,8 +86,9 @@ def test_lstm_memorize_marker(self):
             seed=1,
             verbose=False,
         )
-        em.train_model(Xs[0], Ys[0], Xs[1], Ys[1], n_epochs=10, verbose=False)
-        score = em.score(Xs[2], Ys[2], verbose=False)
+        em.train_model((Xs[0], Ys[0]), dev_data=(Xs[1], Ys[1]),
+                       n_epochs=15, verbose=False)
+        score = em.score((Xs[2], Ys[2]), verbose=False)
```

Comment:
I think there's a broader discussion to be had about what sort of tests best target attention... but for now, could you put in a quick regression test?  I.e. just train the LSTM without and then with attention, and make sure the score isn't worse?